diff --git a/node_modules/react-pdf/dist/esm/Page/TextLayer.js b/node_modules/react-pdf/dist/esm/Page/TextLayer.js
index b28f117..559de44 100644
--- a/node_modules/react-pdf/dist/esm/Page/TextLayer.js
+++ b/node_modules/react-pdf/dist/esm/Page/TextLayer.js
@@ -4,7 +4,7 @@ import React from 'react';
 import { createPortal } from 'react-dom';
 import invariant from 'tiny-invariant';
 import warning from 'warning';
-import * as pdfjs from 'pdfjs-dist';
+import { renderTextLayer, setLayerDimensions } from 'pdfjs-dist';
 import usePageContext from '../shared/hooks/usePageContext.js';
 import useResolver from '../shared/hooks/useResolver.js';
 import { cancelRunningTask } from '../shared/utils.js';
@@ -107,7 +107,7 @@
     }
     function onRenderSuccess() {
         if (textLayer) {
-            const { textContentSource, viewport } = textLayer;
+            const { textContent, viewport } = textLayer;
             const container = textLayerDiv.current;
             const stream = textContent;
             setLayerDimensions(container, viewport);
@@ -115,7 +115,7 @@
                 container,
                 textContent: stream,
                 viewport,
-                textDivs: [],
+                textContentSource: stream,
             }).promise;
             setRunningTask(renderTask);
         }
